src/test/java/com/wikia/webdriver/common/testnglisteners/BrowserAndTestEventListener.java

```diff
@@ -123,6 +123,8 @@ public void afterNavigateTo(String url, WebDriver driver) {
             userOptedOut = true;
           }
 
+          String cmpVersion = "2";
+
           if (userOptedIn) {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Refactor this code to not nest more than 3 if/for/while/switch/try statements. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS134)